reana_client/validation/utils.py

```diff
@@ -51,12 +51,35 @@ def validate_reana_spec(
             f"Verifying REANA specification file... {filepath}",
             msg_type="info",
         )
-        validate_reana_yaml(reana_yaml)
-        display_message(
-            "Valid REANA specification file.",
-            msg_type="success",
-            indented=True,
-        )
+        validation_warnings = validate_reana_yaml(reana_yaml)
+        for warning_key, warning_value in validation_warnings.items():
+            if warning_key == "additional_properties":
+                # warning_values is a list of unexpected properties
+                message = (
+                    f"Unexpected properties found in REANA specification file: "
+                    f"{', '.join(warning_value)}"
+                )
+            else:
+                message = warning_value
+            display_message(
+                message,
+                msg_type="warning",
+                indented=True,
+            )
+        if validation_warnings:
+            display_message(
+                "The REANA specification file is valid, but some warnings were "
+                "found. Please check the warnings above and make sure that the "
+                "REANA specification file is correct.",
```

Comment:
Given that we already print the warnings before, maybe we can shorten this to just say _Please make sure that the REANA specification file is correct_?
Or we could also print something like this:
```
  -> WARNING: Issues found while validating the REANA specification file
  -> WARNING: [... list of warnings ...]
  -> WARNING: Please make sure that the REANA specification file is correct
```

In any case, what I would like to avoid is to print a very long line in the terminal.

What do you think?

Reply:
Okay for shortening the single lines printed in the terminal! I think it is still important to write that the REANA specification file appears valid.